mypy/stubgenc.py

```diff
@@ -832,7 +834,14 @@ def generate_class_stub(self, class_name: str, cls: type, output: list[str]) ->
             bases_str = "(%s)" % ", ".join(bases)
         else:
             bases_str = ""
-        if types or static_properties or rw_properties or methods or ro_properties:
+        if (
+            types
+            or static_properties
+            or rw_properties
+            or methods
+            or ro_properties
+            or class_info.docstring
+        ):
```

Comment:
Is this related to the issue with `Optional`? I think this change (and the one below) are causing the CI failure.